lib/swaggerValidator/ajvSchemaValidator.ts

```diff
@@ -215,6 +215,16 @@ const shouldSkipError = (error: ErrorObject, cxt: SchemaValidateContext) => {
     return true;
   }
 
+  // If a response has property which x-ms-secret value is true in post we can skip this error
+  if (
```

Comment:
Can you check if traffic validation will fail when the schema only has "x-ms-mutability": ["create","update"]?